docker/build.bash

```diff
@@ -31,6 +31,14 @@ then
     exit 1
 fi
 
+# If the user is not in the docker group, provide an informative error message
+if [[ $(grep /etc/group -e "docker") != *"${USER}"* ]]; then
+  >&2 echo -e "Error: The current user '${USER}' is not in the docker group. \
```

Comment:
So I just tested this out locally, and it appears that this is a bit too simplistic a view of docker.  That is, in my setup, my username is *not* in the docker setup, but this script works fine for me.  That may have to do with the fact that I'm using `podman-docker` rather than regular `docker`, but for most practical purposes everything works the same.

So with all of that said, I actually think we shouldn't put it in at this time, given that different setups may have different requirements.

Reply:
> this is a bit too simplistic a view of docker.

I see, you are right. 
I have not heard about `podman-docker` before. It seems interesting. Thanks for sharing!

> So with all of that said, I actually think we shouldn't put it in at this time, given that different setups may have different requirements.

That makes sense. Feel free to close this PR.

Reply:
OK, thanks for the work here anyway!  Closing this.